packages/mgt-components/src/components/mgt-teams-channel-picker/mgt-teams-channel-picker.ts

```diff
@@ -360,17 +401,28 @@ export class MgtTeamsChannelPicker extends MgtTemplatedComponent {
    */
   protected renderSelected() {
     if (!this._selectedItemState) {
-      return html``;
+      return this.renderSearchIcon();
     }
+    let icon: TemplateResult;
+    if (this._selectedItemState.parent.channels) {
+      icon = html`<img
+        class="team-photo"
+        alt="${this._selectedItemState.parent.item.displayName}"
+        src=${this.teamsPhotos[this._selectedItemState.parent.item.id]?.photo} />`;
+    }
+
+    const parentName = this._selectedItemState?.parent?.item?.displayName.trim();
+    const channelName = this._selectedItemState?.item?.displayName.trim();
 
     return html`
-      <li class="selected-team" title=${this._selectedItemState.item.displayName}>
-        <div class="selected-team-name">${this._selectedItemState.parent.item.displayName}</div>
-        <div class="arrow">${getSvg(SvgIcon.TeamSeparator, '#B3B0AD')}</div>
-        ${this._selectedItemState.item.displayName}
-        <div class="search-wrapper">${this.renderSearchIcon()} ${this.renderInput()}</div>
-      </li>
-    `;
+      <fluent-breadcrumb title=${this._selectedItemState.item.displayName}>
+        <fluent-breadcrumb-item>
+          <span slot="start">${icon}</span>
+          <span slot="end" class="team-parent-name">${parentName}</span>
```

Comment:
Shouldn't this just be in the default slot? https://github.com/microsoft/fast/blob/master/packages/web-components/fast-foundation/src/breadcrumb-item/breadcrumb-item.ts for reference on the slots we can use

Reply:
Yes, it can. However, it meant having CSS to align the items(icon and team name). So, I used the slots to just have the alignment out of the box. The separator slot is to use the specified svg in the designs.

Reply:
Could that css leverage `fluent-breadcrumb-item::part(listitem)` to address provide the alignment that is needed?

Reply:
Yes, it can leverage the selector, but you have to write out the CSS for the icon + title alignment. Using the slots in the first `fluent-breadcrumb` just aligns it out of the box.